### ISSUE TITLE
.swiftlint.yml not working

### ISSUE THREAD

**Issue by tspecht** [realm/SwiftLint#249]
Hi guys,

sorry for this probably totally stupid question but I'm not able to get the .swiftlint.yml file to work for excluding rules :( Here's the content of .swiftlint.yml I'm using:

```
disabled_rules: # rule identifiers to exclude from running
  - trailing_whitespace
included: # paths to include during linting. `--path` is ignored if present. takes precendence over `excluded`.
  - MyCodeFolder/Path/
excluded: # paths to ignore during linting. overridden by `included`.
  - Carthage
  - Pods
# parameterized rules can be customized from this configuration file
line_length: 110
# parameterized rules are first parameterized as a warning level, then error level.
type_body_length:
  - 300 # warning
  - 400 # error
# parameterized rules are first parameterized as a warning level, then error level.
variable_name_max_length:
  - 40 # warning
  - 60 # error
# parameterized rules are first parameterized as a warning level, then error level.
variable_name_min_length:
  - 3 # warning
  - 2 # error
```

When I run `swiftlint lint` or `swiftlint lint --config .swiftlint.yml` the file gets not honored ... any ideas?

**Comment by jpsim** [comment]
Your YAML [validates](http://www.yamllint.com/) and when I paste it into SwiftLint's `.swiftlint.yml` it gets parsed properly when I run `swiftlint` from that root.

Are you saying that `Loading configuration from '.swiftlint.yml'` doesn't get printed to the console? Then it must be a path error. Did you perhaps name the file something slightly different (e.g. `.swiftlint.yaml`)?

**Comment by tspecht** [comment]
Yeah, I don't see any log about a configuration being loaded. I checked with `ls -alh` and the file seems to have both correct access rights as well as the correct naming:
`-rw-r--r--   1 myUser  staff   807B Dec  5 16:52 .swiftlint.yml`
In addition I made sure that the file also lies in the very same directory I'm executing `swiftlint lint` from. Is there any way I also could disable rules using command line arguments?

**Comment by tspecht** [comment]
Hmm sorry for the confusion, I guess my brew didn't correctly update swiftlint ... my bad, sorry again.

**Comment by jeanpierrevg** [comment]
Hi, i have the same issue. You fixed it by using brew install swiftlint ? When i try to do that, since I am on OX 10.11.12 it won't install. When I install the swiftlint as a package I can run the swiftlint but it does not take into account the Exclude folders such as Pods ...any ideas?

**Comment by toshi0383** [comment]
You might want to format your code block.

```
```

**Comment by jeanpierrevg** [comment]
Sorry, i am attaching the file for better understanding
[.swiftlint.yml.zip](https://github.com/realm/SwiftLint/files/70323/default.swiftlint.yml.zip)

**Comment by norio-nomura** [comment]
@jeanpierrevg By applying #287, SwiftLint produces an error on using your `.swiftlint.yml`
```
$ swiftlint lint --config .swiftlint.yml
unexpected type doc-end, near "...\n  # Find all the available rules by running:\n "
```
It seems your `.swiftlint.yml` is invalid on following line:
```YAML
  - ...
  # Find all the available rules by running:
```

**Comment by jeanpierrevg** [comment]
@norio-nomura I have tried with v0.5.2 and also cloning the master and also by https://github.com/realm/SwiftLint and git submodule update --init --recursive; make install ....in both cases the $ swiftlint lint --config .swiftlint.yml runs but ignores the Excluded folders so is listing the Pods folder etc ....I am using the configuration format mentioned in the readme notes of https://github.com/realm/SwiftLint ....should i use a different format?

**Comment by jpsim** [comment]
#287 was just merged

**Comment by jeanpierrevg** [comment]
I just did a https://github.com/realm/SwiftLint and thengit submodule update --init --recursive; make install so I must have the latest one. Still the Excluded rules is ignored

**Comment by jeanpierrevg** [comment]
The problem was the whitespaces , seems like copying it from the realm documentation set tabs instead of whitespace

**Comment by jpsim** [comment]
> The problem was the whitespaces , seems like copying it from the realm documentation set tabs instead of whitespace

I don't know what you mean by this. If this is something we should fix, can you elaborate or fix it in a PR? Thanks!

**Comment by jeanpierrevg** [comment]
Sorry, i was not clear....basically I copied the Configuration's piece from https://github.com/realm/SwiftLint into a file , seems like doing that caused that white space were having more than 2 spaces and also i think they seemed to be tabs instead of white spaces ...I saw that somebody reported this on another thread you were responding to as well. So in the end it was just a copy&paste format issue.

**Comment by jpsim** [comment]
Thanks for clearing that up. #287 should make more informative error messages moving forward.

**Comment by jaykarparmar** [comment]
Hello,
I have installed **swiftlint** using [SwiftLint.pkg](https://github.com/realm/SwiftLint/releases/tag/0.39.2) in my system. Now I want to customize the rules so I have added **.swiftlint.yml** in my root directory. But somehow it's not working.
